src/ILCompiler.ReadyToRun/src/IBC/ibcdatareader.cs

```diff
@@ -0,0 +1,724 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Collections;
+using System.Diagnostics;
+
+namespace ILCompiler.IBC
+{
+    public class IBCDataReader
+    {
+        // These fields are only valid during a call to Read()
+        private BinaryReader reader;
```

Comment:
Prefix with underscore?

Reply:
I'm trying to keep that file in sync (or very close to it) with the the closed source variant. I'd rather not make a formatting change like that at this time.

Reply:
I just realized that we have a process for when closed source files are moved to open that we're skipping here: it involves running Policheck and CodeFormatter - is there a reason why we need exemption from the process?

The reformatted files get checked in on the closed source side, same way we did for the hundreds of other files in this repo - that should eliminate the divergence concern.

The problem with "at this time" reasons is that historically we never find time to fix what wasn't fixed at checkin time, hurting readability in the coming years (not following naming conventions for fields definitely hurts my ability to code review this because I can't distinguish them from locals).